cli/cloudproviders/baremetal.py

```diff
@@ -100,8 +101,11 @@ def _teuthology_lock(
         # Check for --machine-type parameter
         cmd += f" --machine-type {machine_type}" if machine_type else ""
 
+        # Get all node details
+        cmd += " --all" if all else ""
+
         # Set teuthology environment
-        cmd = f"{self._env}/{cmd}"
+        cmd = f"/{self._env}/{cmd}"
```

Comment:
Need to update [1] to use `rstrip` instead of `strip`
[1] https://github.com/red-hat-storage/cephci/pull/3541/files#diff-54f72a2e5c85d8ca16775b7b25105f174af55f6db2ca31dbff2adf1a994e40b7R35